client/reader/components/icons/x-icon.jsx

```diff
@@ -0,0 +1,15 @@
+export default function ReaderXIcon( { iconSize } ) {
+	return (
+		<svg
+			key="x"
+			fill="none"
+			className="reader-x"
+			viewBox="0 0 1200 1227"
```

Comment:
The value seems unusually large 🤔

Reply:
I downloaded the SVG right from the [X brand toolkit](https://about.x.com/en/who-we-are/brand-toolkit). It's a very strange aspect ratio that doesn't scale to easy, round numbers 🧐 Here's an example:

<img width="688" alt="Screenshot 2025-02-17 at 17 32 23" src="https://github.com/user-attachments/assets/ccc07764-a80c-4fdf-8141-c9be8b71295d" />

I personally don't see a compelling reason to change the official values, though I'm happy to discuss further! Let me know what you think.